REVIEWING.md

```diff
@@ -110,28 +110,13 @@ of Porter. When you are asked to cut a new release, here is the process:
     ```
 
 1. Generate some release notes and put them into the release on GitHub.
-    The following command gives you a list of all the merged pull requests:
-
-    ```
-    git log --oneline OLDVERSION..NEWVERSION
-    ```
-
-    You need to go through that and make a bulleted list of features
-    and fixes with the PR titles and links to the PR. If you come up with an
-    easier way of doing this, please submit a PR to update these instructions. 😅
-
-    ```
-    # Features
-    * PR TITLE (#PR NUMBER)
-
-    # Fixes
-    * PR TITLE (#PR NUMBER)
-
-    # Install or Upgrade
```

Comment:
Can you keep this part too? I want to make sure that if someone is looking at a release, they can quickly find the instructions to install it